client/src/components/RecentReviewList/RecentReviewListEntry.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
```

Comment:
maybe you haven't built this one out yet or just want to keep it separate but being only one div it doesn't really need its own file?